src/utils/callbackSet.ts

```diff
@@ -0,0 +1,45 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+  export module Utils {
+    /**
+     * A set of callbacks which can be all invoked at once.
+     * Each callback exists at most once in the set (based on reference equality).
+     * Ideally, all callbacks should have the same signature.
```

Comment:
This is not an exposed API point (or will not be), hence I prefer clarity, but if you want, I can remove some of that information